examples/datamart-from-FHIR-demo/example_schema.yaml

```diff
@@ -0,0 +1,103 @@
+---
+table_1:
```

Comment:
nit: at this level it's more of a collection of tables than a table right?

Reply:
I could be wrong, but I believe the top level is the table.

Reply:
For the following schema:
```
table_1:
  Patient:
    Patient ID:
      fhir_path: Patient.id
      include_nulls: false
      include_unknowns: false
      selection_criteria: first
      new_name: patient_id
  Observation:
      Observation ID:
        fhir_path: Observation.id
        include_nulls: false
        include_unknowns: false
        selection_criteria: first
        new_name: observation_id
```
We'll end up with a directory structure of:
ouput_path
 |- table_1
   |- Patient.parquet
   |- Observation.parquet

I was kinda equating each parquet file as a table.  I may be misinterpreting.  I don't think this is a blocking issue, so I'm going to go ahead and approve, but may be worth getting on the same page with terminology at some point.  Thanks!

Reply:
Yes, this results in a `table_1` directory containing `Patient.parquet` and `Observation.parquet`. This was an intentional change I made, but didn't document well (I'll address that now). My understanding of the current schema format is that the top level represents tables within the schema. The motivation for this change is to be able to handle schemas that specify tables containing fields from multiple resources types. We haven't yet tackled exactly how we want to combine data across resources types into single tables. For the time being this allows us to process any schema a user might submit by creating resource type specific "sub-tables" for each table specified in a schema. 

From here I see a few possible paths:
1. After the sub-tables are written, read them them back in and perform joins between them to make the desired table.
2. Same as above, but keep the sub tables in memory.
3. Get more sophisticated at interpreting the schema config files and/or add additional information to them that would allow us to execute searches using more advanced features like `_include` and `_revinclude`. https://build.fhir.org/search.html#revinclude

Reply:
+1 to what Dan is saying. I think for the purposes of the demo, and for MVP purposes, this is sufficient as the ECR schema only reads fields from a single resource for any given table. Moving forward, exploring the three options Dan has proposed will be a good next step.